flow/include/flow/DebugTrace.h

```diff
@@ -26,4 +26,7 @@
 constexpr bool debugLogTraces = false;
 #define DebugLogTraceEvent(...) DebugTraceEvent(debugLogTraces, __VA_ARGS__)
 
+constexpr bool debugRelocationTraces = false;
+#define DebugRelocationTraceEvent(...) DebugTraceEvent(debugRelocationTraces, __VA_ARGS__)
```

Comment:
Just a thought - it would be nice to enable this dynamically. Maybe an actor that polls a system key that can be used to turn this on and off? I don't know yet about dynamic knobs, but maybe that mechanism?

Reply:
I'll create an issue to track this